src/drift.core/model/ApplicationGroup.cs

```diff
@@ -11,17 +11,28 @@
 namespace Rangers.Antidrift.Drift.Core
 {
     using System.Collections.Generic;
+    using System.Linq;
 
     public class ApplicationGroup
     {
         public string Descriptor { get; set; }
 
         public bool IsSpecial { get; set; }
 
+        public string Name { get; set; }
+
+        public IList<Namespace> Namespaces { get; set; } = new List<Namespace>();
+
         public string[] Members { get; set; } = System.Array.Empty<string>();
 
-        public string Name { get; set; }
+        public ApplicationGroup Expand(TeamProject teamProject)
+        {
+            var result = new ApplicationGroup();
```

Comment:
Can be simplified to:

            return new ApplicationGroup
            {
                Name = this.Name.Expand(teamProject),
                Namespaces = this.Namespaces.Select(n => n.Expand(teamProject)).ToList(),
                Members = this.Members.Select(m => m.Expand(teamProject)).ToArray(),
            };